src/main/resources/view/PersonDetailView.fxml

```diff
@@ -1,14 +1,123 @@
+                            <image>
+                                <Image url="@/images/user_outline_32.png" />
+                            </image>
+                        </ImageView>
+                        <Text styleClass="detail-contents"> Relationship </Text>
+                    </HBox>
+                    <Label fx:id="relationship" styleClass="detail-contents" text="\$relationship"/>
+                </HBox>
+
+                <HBox spacing="15" alignment="CENTER_LEFT">
+                    <HBox spacing="15" prefWidth="150" alignment="CENTER_LEFT">
+                        <ImageView>
+                            <image>
+                                <Image url="@/images/phone_outline_32.png" />
+                            </image>
+                        </ImageView>
+                        <Text styleClass="detail-contents"> Phone </Text>
+                    </HBox>
+                    <Label fx:id="phone" styleClass="detail-contents" text="\$phone" />
+                </HBox>
+
+                <HBox spacing="15" alignment="CENTER_LEFT">
+                    <HBox spacing="15" prefWidth="150" alignment="CENTER_LEFT">
+                        <ImageView>
+                            <image>
+                                <Image url="@/images/mail_outline_32.png" />
+                            </image>
+                        </ImageView>
+                        <Text styleClass="detail-contents"> Email </Text>
+                    </HBox>
+                    <Label fx:id="email" styleClass="detail-contents" text="\$email" />
+                </HBox>
+            </VBox>
+
+            <!-- Upcoming Events Section -->
+            <VBox spacing="25">
+                <Text styleClass="detail-sub-header"> Upcoming Events </Text>
+
+                <!-- GridPane for displaying events -->
+                <GridPane fx:id="gridPane" hgap="10" vgap="15" VBox.vgrow="ALWAYS">
+                    <!-- Header Row -->
+                    <columnConstraints>
+                        <ColumnConstraints percentWidth="100"/>
+                        <ColumnConstraints percentWidth="80"/>
+                        <ColumnConstraints percentWidth="50"/>
+                    </columnConstraints>
+
+                    <children>
+                        <Text styleClass="detail-grid-header" text="Name" GridPane.columnIndex="0" />
+                        <Text styleClass="detail-grid-header" text="Date" GridPane.columnIndex="1" />
+                        <Text styleClass="detail-grid-header" text="Status" GridPane.columnIndex="2" />
+                    </children>
+
+                    <!-- Separator Line -->
+                    <children>
+                        <Region GridPane.columnIndex="0" GridPane.rowIndex="1" GridPane.columnSpan="3"
+                                style="-fx-background-color: #D3D3D3; -fx-min-height: 1px;"/>
+                    </children>
+
+                    <!-- Data Row (Placeholder) -->
```

Comment:
Do we need the placeholder data or can we remove it?

Reply:
I'll remove it